source/chapters/effects.rst

```diff
@@ -61,6 +61,9 @@ in the unit, there will be comb filtering which is probably not desired. For
 more background information, read `Equalizers and Phase Shift
 <http://ethanwiner.com/EQPhase.html>`_.
 
+The shortcut to quickly activate/disable any effect unit for Deck 1 is 5 and 0 for Deck 2 
+in an English keyboard layout.
```

Comment:
Those shortcuts control the single Quick Effects (green) which are located in the mixer
![image](https://user-images.githubusercontent.com/5934199/95658740-edb53e00-0b1c-11eb-96ae-0d2ce10028e3.png)
I suggest you remove this change. Unfortunately there's no mention of the Quick Effects, yet (just created https://github.com/mixxxdj/manual/issues/141 to cover that)

Reply:
> Those shortcuts control the single Quick Effects (green) which are located in the mixer
> ![image](https://user-images.githubusercontent.com/5934199/95658740-edb53e00-0b1c-11eb-96ae-0d2ce10028e3.png)
> I suggest you remove this change. Unfortunately there's no mention of the Quick Effects, yet (just created #141 to cover that)

Change has been removed

Reply:
> Those shortcuts control the single Quick Effects (green) which are located in the mixer
> ![image](https://user-images.githubusercontent.com/5934199/95658740-edb53e00-0b1c-11eb-96ae-0d2ce10028e3.png)
> I suggest you remove this change. Unfortunately there's no mention of the Quick Effects, yet (just created #141 to cover that)

done!